my-app/src/Components/WeatherDay/WeatherDay.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export default function WeatherDay(props) {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [message, setMessage] = React.useState('More details...');
+
+  const handleExpand = () => {
+    setIsOpen(true);
+    setMessage('Show less');
+  };
+
+  const handleCollapse = () => {
+    setIsOpen(false);
+    setMessage('More details...');
+  };
+  return (
+    <>
```

Comment:
` <>` empty tags are not needed if you have a common tag that wraps all the markup of the component